doc/specifications/api/README.md

```diff
@@ -0,0 +1,17 @@
+# API
+
+API in eZ context refers to its Public API's. This folder covering specifically the PHP API, and specifications for
+REST API can be found in `doc/specifications/rest` folder.
```

Comment:
If I understood the meaning right, I'd make it more clear: 
> This folder covers the PHP API specifically. Specifications for REST API can be found in `doc/specifications/rest` folder. 

(btw. folder can be linked in markdown).

cc @DominikaK

Reply:
ok. is the suggested change ok with you as well @DominikaK ?

Reply:
Sounds good to me.